backend/src/utils/resourceUtils.ts

```diff
@@ -677,8 +677,11 @@ export const cleanupGPU = async (fastify: KubeFastifyInstance): Promise<void> =>
     ) {
       // if gpu detected on cluster, create our default migrated-gpu
       const acceleratorDetected = await getDetectedAccelerators(fastify);
+      const hasNvidiaNodes = Object.values(acceleratorDetected.nodeInfo).some((nodeInfo) =>
+        nodeInfo.architecture.includes('nvidia'),
+      );
```

Comment:
you should be use the acceleratorDetected.total object to get the available accelerators and if one of those keys is equal to `nvidia.com/gpu` then you can say that there exists a nvidia accelerator.

The reason for this is that we are creating a resource that has `identifier: 'nvidia.com/gpu'` so we should check directly that the resource exists in the node, not that the node is of some type.